db/migrate/20160502145133_rename_technology_ids_to_keys.rb

```diff
@@ -0,0 +1,10 @@
+class RenameTechnologyIdsToKeys < ActiveRecord::Migration
+  def change
+    add_column :importable_attributes, :technology_key, :string, after: :technology_id
+
+    ImportableAttribute.all.each do |ia|
```

Comment:
```
uninitialized constant RenameTechnologyIdsToKeys::ImportableAttribute
moses/db/migrate/20160502145133_rename_technology_ids_to_keys.rb:5:in `change'
NameError: uninitialized constant RenameTechnologyIdsToKeys::ImportableAttribute
```

`ImportableAttribute` class has been removed so this migration won't work anymore. I'm guessing it can just be removed?